server/build/gitlab-dc.common.yml

```diff
@@ -1,29 +1,44 @@
 version: '2.4'
 services:
   minio:
-    image: ${CI_REGISTRY}/mattermost/ci/images/minio:RELEASE.2019-10-11T00-38-09Z-1
+    image: public.ecr.aws/mattermostqa/minio:RELEASE.2019-10-11T00-38-09Z-1
```

Comment:
Where is the CI for those ? How did we upload them to ECR and what is the rationale around it ? 
cc: @mattermost/core-build-engineers

Reply:
But for reference: the ECR repo is managed by [this script](https://git.internal.mattermost.com/qa/aws-test/-/blob/main/scripts/ecrpublic_upload.sh), which mirrors [these images](https://git.internal.mattermost.com/qa/aws-test/-/blob/main/terraform/us-east-1/generic/ecrpublic/terraform.tfvars.json) from Docker Hub to ECR. No CI is currently built around it

Reply:
+1 to `mattermostdevelopment/*` and would be great to have CI when adding/bumping images.

Reply:
This will be tackled separately, as part of [this ticket](https://mattermost.atlassian.net/browse/CLD-5703)